plugins/inputs/logfile/tailersrc_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"log"
```

Comment:
Are the changes in this file related to code cov is so how?

Reply:
No, this is a flakey test that uses timers to test asynchronous code... I'm almost tempted to just rip it out but for now just increased the timers instead.